flycheck.el

```diff
@@ -6068,6 +6069,27 @@ See URL `http://www.foodcritic.io'."
        ;; http://matschaffer.github.io/knife-solo/#label-Init+command
        (locate-dominating-file parent-dir "cookbooks")))))
 
+(flycheck-define-checker chicken-scheme
+  "A CHICKEN Scheme syntax checker using CHICKEN compiler `csc'.
+
+See URL `http://call-cc.org/'."
+  :command ("csc" "-A" "-local" source-inplace)
```

Comment:
Do we need `source-inplace` here?  If so please add a comment that explains why we're using `source-inplace` instead of `source` here.